src/syzygy/tbprobe.cpp

```diff
@@ -1593,7 +1593,7 @@ bool Tablebases::root_probe_wdl(Position& pos, Search::RootMoves& rootMoves) {
 
     static const int WDL_to_rank[] = { -MAX_DTZ, -MAX_DTZ + 101, 0, MAX_DTZ - 101, MAX_DTZ };
```

Comment:
Maybe we could also change this to constexpr  while in here.

Reply:
It's unrelated to the fixed problem, so I think this belongs into a separate pull request.